criu/uffd.c

```diff
@@ -281,7 +281,7 @@ int uffd_open(int flags, unsigned long *features, int *err)
 	}
 
 	if (uffdio_api.api != UFFD_API) {
-		pr_err("Incompatible uffd API: expected %Lu, got %Lu\n", UFFD_API, uffdio_api.api);
+		pr_err("Incompatible uffd API: expected %llu, got %llu\n", UFFD_API, uffdio_api.api);
```

Comment:
should we use PRI64 in this case?

Reply:
Ok, I see:
```
typedef u64			uint64_t;
```
https://github.com/torvalds/linux/blob/master/include/linux/types.h#L107
```
typedef __u64 u64;
```
https://github.com/torvalds/linux/blob/master/include/asm-generic/int-ll64.h#L23

Probably it's enough proof.

Reply:
Heh, but tests don't agree =)
```
In file included from criu/include/criu-log.h:23,
                 from criu/uffd.c:22:
criu/uffd.c: In function ‘uffd_open’:
criu/include/log.h:49:52: error: format ‘%lu’ expects argument of type ‘long unsigned int’, but argument 5 has type ‘long long unsigned int’ [-Werror=format=]
   49 | #define pr_err(fmt, ...) print_on_level(LOG_ERROR, "Error (%s:%d): " LOG_PREFIX fmt, __FILE__, __LINE__, ##__VA_ARGS__)
      |                                                    ^~~~~~~~~~~~~~~~~
criu/uffd.c:284:3: note: in expansion of macro ‘pr_err’
  284 |   pr_err("Incompatible uffd API: expected %" PRIu64 ", got %" PRIu64 "\n", UFFD_API, uffdio_api.api);
      |   ^~~~~~
```
So I'd rather revert it to `%llu` variant.